src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1020,7 +1020,7 @@
     <value>An enumeration's values should not start with the type name of the enumeration.</value>
   </data>
   <data name="DoNotPrefixEnumValuesWithTypeNameMessage" xml:space="preserve">
-    <value>Do not prefix enum values with the name of the enum type '{0}'. </value>
+    <value>Do not prefix enum values with the name of the enum type '{0}'. .</value>
```

Comment:
This should be fixed. We should update the analyzer to flag leading and trailing spaces for all 3 kind of strings.

Reply:
See #3958